k8s/k8s-descriptor-template-verify.yaml

```diff
@@ -68,8 +68,8 @@ spec:
             value: prod
           - name: WIKIA_DATACENTER
             value: sjc
-          - name: INTERNAL_CACHE_HOST
-            value: prod.icache.service.sjc.consul
+          - name: MW_INTERNAL_HOST
+            value: mediawiki-verify
```

Comment:
There is no verify on k8s

Reply:
so should we sunset verify env on mobile-wiki?

Reply:
I believe so